pre_vs_berkeley.md

```diff
@@ -0,0 +1,440 @@
+# Merging the Berkeley schema into the NMDC schema
+
+_Corresponding to a jump from major version 10 to 11_
+
+This major refactoring increases the number of **classes** in the schema from 70 to 75, but decreases the number of root
+classes from 19 to 17 due to better organization.
+
+| Removed legacy class                | New/Replacement class from Berkeley schema | Selected legacy parents   |
```

Comment:
When I saw this table, I wondered "what is this?" I suggest adding a sentence above the table, saying "Here's a table of..."